google/cloud/bigtable/testing/table_integration_test.cc

```diff
@@ -105,7 +110,43 @@ void TableIntegrationTest::SetUp() {
       TableTestEnvironment::project_id(), TableTestEnvironment::instance_id(),
       ClientOptions());
 
-  ASSERT_STATUS_OK(table_admin_->DropAllRows(TableTestEnvironment::table_id()));
+  // We cannot use `DropAllRows()` to cleanup the table because the integration
+  // tests against production sometimes use all the "'DropRowRangeGroup' and
+  // limit 'USER-100s'" quota. Instead we delete the rows by using a
```

Comment:
I assume you meant the sentence to run into the code.  Just repeating "BulkMutation", or saying "bulk mutation", is probably clearer.

Reply:
I thought I had push the fix for that, sorry.